src/cms/templates/events/event_form.html

```diff
@@ -399,89 +399,92 @@ <h3 class="font-bold my-3">{% trans 'Where' %}</h3>
+                    editor.ui.registry.addMenuItem('pinicon', {
+                        text: '{% trans 'Location' %}',
+                        icon: 'pin_icon',
+                        onAction: function () {
+                            editor.insertContent('<img src="{% static "svg/pin.svg" %}" style="width:1.5em">');
+                        }
+                    });
+                    editor.ui.registry.addMenuItem('wwwicon', {
+                        text: '{% trans 'Link' %}',
+                        icon: 'www_icon',
+                        onAction: function () {
+                            editor.insertContent('<img src="{% static "svg/world-wide-web.svg" %}" style="width:1.5em">');
+                        }
+                    });
+                    editor.ui.registry.addMenuItem('callicon', {
+                        text: '{% trans 'Phone' %}',
+                        icon: 'call_icon',
+                        onAction: function () {
+                            editor.insertContent('<img src="{% static "svg/call.svg" %}" style="width:1.5em">');
+                        }
+                    });
+                    editor.ui.registry.addMenuItem('clockicon', {
+                        text: '{% trans 'Opening Hours' %}',
+                        icon: 'clock_icon',
+                        onAction: function () {
+                            editor.insertContent('<img src="{% static "svg/clock.svg" %}" style="width:1.5em">');
+                        }
+                    });
+                    editor.ui.registry.addMenuItem('aticon', {
+                        text: '{% trans 'Email' %}',
+                        icon: 'email_icon',
+                        onAction: function () {
+                            editor.insertContent('<img src="{% static "svg/at.svg" %}" style="width:1.5em">');
+                        }
+                    });
+                    editor.ui.registry.addMenuItem('ideaicon', {
+                        text: '{% trans 'Hint' %}',
+                        icon: 'idea_icon',
+                        onAction: function () {
+                            editor.insertContent('<img src="{% static "svg/idea.svg" %}" style="width:1.5em">');
+                        }
+                    });
+                }{% has_perm 'cms.edit_events' request.user as can_edit_event %}{% if not can_edit_event or event_form.instance.id and event_form.instance.archived %},
             readonly : 1{% endif %}
-    });
+            });
 
-    custom_file_field();
-}, false);
-</script>
+            custom_file_field();
+        }, false);
+    </script>
```

Comment:
This diff seems very broken, but all you added was a bit of indentation, right? :sweat_smile:

Reply:
Yes, I included my own script above and wanted to have proper indentation. This looks weird in the generated HTML source if you look it up in the browser but imho it looks better in the IDE (which was more important to me). If you don't agree, I can of course easily remove the indentation.

Reply:
No, the indentation is fine, I just wondered whether I missed some other changes there.